_pages/en_US/faq.md

```diff
@@ -25,7 +25,7 @@ Playing cartridge dumps on the console requires the use of a flashcart or nds-bo
 ### How can I restore my NAND backup if I cannot boot the console?
 You can follow Gadorach's [hardmodding guide](https://web.archive.org/web/20151102221503/https://gbatemp.net/threads/dsi-downgrading-the-complete-guide.393682/) to hardmod your DSi. Previous soldering experience is required.
 
-### How do I update my Nintendo DSi Homebrew?
+### How do I update my homebrew?
 - **Unlaunch** - Follow the instructions on the [Installing Unlaunch](/installing-unlaunch) page
 - **hiyaCFW** - Replace `hiya.dsi` on the root of the SD card
 - **TWiLight Menu++** - Follow Section 1b of [Launching the Exploit](launching-the-exploit#twilight-menu)
```

Comment:
shouldn't we link the official wiki for TWiLight Menu++ here?

Reply:
Already covered by Evie's review, I have to commit it separately